input/tar2/goconvey.goconvey

```diff
@@ -0,0 +1,2 @@
+#-timeout=30s
+-short
```

Comment:
... that's odd.  Must have lost it on a rebase.  I'll lose this too, to match, and add it back in separately later.